pedant/decorators.py

```diff
@@ -206,8 +235,13 @@ def log_render(*args, **kwargs):
         return ''
 
     patch_base = patch.object(VariableNode, 'render', log_render)
-    patch_debug = patch.object(DebugVariableNode, 'render', log_debug_render)
-    return lambda f: patch_base(patch_debug(f))
+    patch_all = patch_base
+    if django.VERSION[:3] < (1, 9):
```

Comment:
Ah, good point. For the version tuples I'm using, the comparisons should work fine. I was initially concerned about the 4th tuple item having a value that could be one of `['alpha', 'beta', 'rc', 'final']`, where lexicographic comparisons could be wrong ("release candidate" should be smaller than "final"). However, because I don't use the 4th item, the comparisons should be equivalent. Updated in 5236cbedab08e7196f6b49f19d4ee7b044447697